Content.Shared/Nyanotrasen/Item/PseudoItem/SharedPseudoItemSystem.cs

```diff
@@ -0,0 +1,155 @@
+using Content.Shared.DoAfter;
+using Content.Shared.Hands;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Interaction.Events;
+using Content.Shared.Item;
+using Content.Shared.Item.PseudoItem;
+using Content.Shared.Storage;
+using Content.Shared.Storage.EntitySystems;
+using Content.Shared.Tag;
+using Content.Shared.Verbs;
+using Robust.Shared.Containers;
+
+namespace Content.Shared.Nyanotrasen.Item.PseudoItem;
+
+public class SharedPseudoItemSystem : EntitySystem
+{
+    [Dependency] private readonly SharedStorageSystem _storageSystem = default!;
+    [Dependency] private readonly SharedItemSystem _itemSystem = default!;
+    [Dependency] private readonly SharedDoAfterSystem _doAfter = default!;
+    [Dependency] private readonly TagSystem _tagSystem = default!;
+
+    [ValidatePrototypeId<TagPrototype>]
+    private const string PreventTag = "PreventLabel";
```

Comment:
What does this tag do?

Reply:
Stops you from applying labels using the hand labeler. Once people found this out people abused the crap out of it